ads/README.md

```diff
@@ -39,6 +39,7 @@ We will provide the following information to the ad:
 - `window.context.pageViewId` contains a relatively low entropy id that is the same for all ads shown on a page.
 - [ad viewability](#ad-viewability)
 - `window.context.startTime` contains the time at which processing of the amp-ad element started.
+- `window.context.data.container` contains the name of ad container element if exist.
```

Comment:
You raised a good point. As discussed offline, we should refactor the code a bit to fit in the current structure.